src/main/resources/application.yml

```diff
@@ -1,18 +1,28 @@
 server:
   port: 8080
   contextPath: /api

```

Comment:
what are these ^M in the diff? are those non-printable characters that we should be worried about?